README.md

```diff
@@ -61,13 +61,15 @@ firectl \
 Getting Started on AWS
 ---
 
-- Create an `i3.metal` instance using Amazon Linux 2
+- Create an `i3.metal` instance using Ubuntu 18.xx
```

Comment:
We should specify more precisely which release should be used.  Also, what's the reason for switching from Amazon Linux 2 to Ubuntu.

Reply:
This was after a discussion with @nmeyerhans a couple of weeks back. Also, Ubuntu is a more likely O/S to be chosen by developers, particularly if we're going across multiple clouds.